edi_utm_oca/models/edi_exchange_type.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class EDIExchangeType(models.Model):
+    _inherit = "edi.exchange.type"
+
+    edi_source_id = fields.Many2one(
+        "utm.source",
+        default=lambda self: self._default_edi_source_id(),
+        help="edi source to set on the sale order",
```

Comment:
Could you help to rewrite a `help` in generic way, because maybe this one can apply on stock, product, purchase,... if needed in the future